tests/Form/Type/Operator/StringOperatorTypeTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Form\Type\Operator;
+
+use Sonata\AdminBundle\Form\Type\Operator\StringOperatorType;
+use Symfony\Component\Form\Test\TypeTestCase;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class StringOperatorTypeTest extends TypeTestCase
+{
+    protected $type;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    public function testConfigureOptions()
+    {
+        $formType = new StringOperatorType();
+        $optionsResolver = new OptionsResolver();
+
+        $formType->configureOptions($optionsResolver);
+        $choices = ['choices' => [
+            'label_type_contains' => StringOperatorType::TYPE_CONTAINS,
+            'label_type_not_contains' => StringOperatorType::TYPE_NOT_CONTAINS,
+            'label_type_equals' => StringOperatorType::TYPE_EQUAL,
+            'label_type_starts_with' => StringOperatorType::TYPE_STARTS_WITH,
+            'label_type_ends_with' => StringOperatorType::TYPE_ENDS_WITH,
+        ]];
+
```

Comment:
You forgot to call
```
$options = $optionResolver->resolve();
```

Reply:
I made changes. Not sure if that's right way to do it.

Reply:
No. I gave you the right thing to do. Just adding the line as I gave you in the place I gave you.

Did you look at the other tests ? https://github.com/sonata-project/SonataAdminBundle/blob/3.x/tests/Form/Type/LegacyAdminTypeTest.php#L56-L58

Reply:
Fixed. I hope it's okay. Thanks :)